package.json

```diff
@@ -28,13 +28,16 @@
   "dependencies": {
     "aws-auto-auth": "^0.1.1",
     "aws-sdk": "^2.1.31",
+    "commander": "^2.8.1",
     "csv-to-vrt": "0.0.2",
+    "end-of-stream": "^1.1.0",
     "fs-extra": "^0.18.4",
-    "hyperquest": "^1.2.0",
-    "s3-upload-stream": "^1.0.7",
-    "unzip": "^0.1.11"
-  },
-  "devDependencies": {
-    "tape": "^4.0.0"
+    "ftp": "^0.3.10",
```

Comment:
I agree re: dev dependencies. In the later PRs I've actually moved everything to plain dependencies and I've stopped shrinkwrapping (which is really a premature optimization imo until development settles down. Having to reshrinkwrap for every new dependency kinda clutters to commits and is another thing to forget). Once I start shrinkwrapping again, I'll set up a pre-commit hook that shrinkwraps automatically so I feel minimum friction.

The reason it's done in the first place is to have exact dependencies so we don't end up with untested broken code due to someone pushed a bad 1.x.x version